mt940/models.py

```diff
@@ -381,7 +381,7 @@ def parse(self, data):
                     self.transactions.append(transaction)
                 else:
                     transaction.data.update(result)
-            elif tag.scope is Transaction and self.transactions:
```

Comment:
I need to parse the same tag (NS) both in Transaction and Transactions scope. As I see it, the current code has a one to one mapping between tags and Tag classes, and the Tag class can be in just one scope.